scripts/helper.sh

```diff
@@ -0,0 +1,213 @@
+#!/bin/bash
+# Global helper functions for WAVS middleware scripts
+# This file should be sourced by other scripts, not run directly
+
+set -o errexit -o nounset -o pipefail
+
+export FOUNDRY_DISABLE_NIGHTLY_WARNING=1
+
+# Parse command line arguments in key=value format
+# Usage: parse_args "$@"
+# Sets variables based on key=value pairs
+parse_args() {
+    for arg in "$@"; do
+        if [[ $arg == *"="* ]]; then
+            key="${arg%%=*}"
+            value="${arg#*=}"
+            # Export the variable so it's available to the calling script
+            export "$key"="$value"
+        fi
+    done
+}
+
+# Check if a required environment variable is set
+# Usage: check_env_var "VARIABLE_NAME" "$VARIABLE_VALUE"
+check_env_var() {
+    local var_name="$1"
+    local var_value="$2"
+    if [ -z "$var_value" ]; then
+        handle_error "$var_name is not set in the environment variables"
+    else
+        export "$var_name"="$var_value"
+        echo "Using value for $var_name: $var_value"
+    fi
+}
+
+# Check if a required parameter is set (either from env or command line)
+# Usage: check_param "PARAM_NAME" "$PARAM_VALUE"
+check_param() {
+    local param_name="$1"
+    local param_value="$2"
+    if [ -z "$param_value" ]; then
+        handle_error "$param_name is not set (check environment variables or command line parameters)"
+    else
+        export "$param_name"="$param_value"
```

Comment:
like this script. and then explicitly setting this if it read the value from some default (`${DEPLOY:-LOCAL}`) is a great idea